src/partials/sale.html

```diff
@@ -0,0 +1,127 @@
+<section class="section">
+    <div class="container">
+
+        <h2 class="sale-header">Sale</h2>
+        
+<div class="sale-container">
+    
+            <div class="sale-images">
+                <ul class="sale-side-image-list">
+                    <li class="sale-side-image-item">
+                        <picture>
+                            <source srcset="../img/sale/salefirstdeskone.webp 1x, ../img/sale/salefirstdesktwo.webp 2x"  width="132px" height="132px"
```

Comment:
change all photos ways to ./img
only one dot 
width and height - only for <img> size for decktop